docs/requirements.stories.mdx

```diff
@@ -0,0 +1,87 @@
+- [Modern Browsers](https://lit.dev/docs/tools/requirements/#building-for-modern-browsers)
+- [Legacy Browsers](https://lit.dev/docs/tools/requirements/#building-for-legacy-browsers)
+
+According to **caniuse.com**, Baklava's browser support is more than **95%**. ([web components](https://caniuse.com/?search=web%20components), [esm](https://caniuse.com/?search=esm)).
+The remaining 5% can be supported by compiling javascript in build time and adding polyfills. You will have an idea about how you can do it in the following sections.
+
+### Modern Browser Support
+| Browser |  Supports ES2019 & web components |
+|---------|------|
+| Chrome  |  >=73  |
+| Firefox |  >=12.1 |
+| Safari  |  >=63  |
+| Edge    |  >=79  |
+
+### Legacy Browser Support
+Supporting older browsers requires some steps described by the Lit documentation.
+- Compiling modern JavaScript syntax to ES5.
+- Transforming ES modules to another module system.
+- Loading polyfills.
+
+For the compiling and transforming, you can install Baklava as an npm package and transpile it during the build time by using Babel.
+[Here](https://medium.com/@sivaraj-v/basic-webpack-4-and-es5-to-es6-transpiler-using-babel-dc66e72c86c6) you can find an example about this.
+
+| Browser |  Compile JS	| Compile JS & load polyfills |
+|---------|------|-----|
+| Chrome  |  67-79	  | <67 |
+| Firefox |  10-12	 | <10|
+| Safari  |  63-71  | <63|
+| Edge    | 79  | - |
+| Edge Classic   | - | <=18 |
+| Internet Explorer   | -  | 11 |
+
+## Pollyfills
+
+Polyfills are piece of code that allows web developers to use features that are not yet supported by some browsers. There is a [polyfills package](https://github.com/webcomponents/polyfills) created and maintained by [Web Components Organization](webcomponents.org) that you can find almost all necessary polyfills.
+
+Some common examples of polyfills for web components are:
+- Custom Elements API: As mentioned before, older browsers do not have native support for the Custom Elements API, which is required to define custom elements. The `custom-elements-es5-adapter.js`` polyfill can be used to support custom elements in these older browsers.
+- Shadow DOM: As previously explained, the Shadow DOM allows web components to have their own encapsulated styles and layout. However, older browsers do not have native support for the Shadow DOM. The `shadydom.min.js` and `shadycss.min.js` polyfills can be used to support the Shadow DOM in older browsers.
+- HTML Templates: The template element is used to define the template for a web component. However, older browsers do not have native support for this element. The `template.js` polyfill can be used to support the template element in older browsers.
+- Babel Polyfill: This is a widely used polyfill that implements a large subset of the ECMAScript 6 standard, including support for modules. You can import it on your code with import `@babel/polyfill`.
```

Comment:
[Their documentation says](https://babeljs.io/docs/en/babel-polyfill) this is deprecated and they suggest to use [core-js](https://github.com/zloirock/core-js).